server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -452,10 +457,6 @@ export class ScribeLambda implements IPartitionLambda {
 			...getLumberBaseProperties(this.documentId, this.tenantId),
 			checkpointReason: reason,
 			lastOffset: this.lastOffset,
-			scribeCheckpointOffset: this.checkpointInfo.currentCheckpointMessage?.offset,
```

Comment:
Looks like some of the recent logs that were added are getting removed in this PR. Maybe pull from the latest?

Reply:
Ah this was likely a merge conflict mistake. Thanks!